frontend/lib/api/form/pollingstation/pollingStationUtils.ts

```diff
@@ -237,3 +237,60 @@ export function isFormSectionEmpty(section: FormSection, values: PollingStationV
       return true;
   }
 }
+
+export type PollingStationFormSectionStatus = "empty" | "unaccepted-warnings" | "accepted-warnings" | "errors";
+export type PollingStationSummary = {
+  countsAddUp: boolean;
+  hasBlocks: boolean;
+  hasWarnings: boolean;
+  hasErrors: boolean;
+  notableFormSections: {
+    status: PollingStationFormSectionStatus;
+    title?: string;
+    formSection: FormSection;
+  }[];
+};
+
+export function getPollingStationSummary(formState: FormState, values: PollingStationValues): PollingStationSummary {
+  const result: PollingStationSummary = {
+    countsAddUp: true,
+    hasBlocks: false,
+    hasWarnings: false,
+    hasErrors: false,
+    notableFormSections: [],
+  };
+
+  Object.values(formState.sections)
+    .filter((section) => section.id !== "save")
+    .sort(sortFormSections)
+    .forEach((section) => {
+      if (section.errors.length > 0) {
+        result.notableFormSections.push({ status: "errors", formSection: section });
+        result.countsAddUp = false;
+        result.hasBlocks = true;
+        result.hasErrors = true;
+      } else if (section.warnings.length > 0) {
+        result.hasWarnings = true;
+        if (section.ignoreWarnings) {
+          result.notableFormSections.push({ status: "accepted-warnings", formSection: section });
+        } else {
+          result.notableFormSections.push({ status: "unaccepted-warnings", formSection: section });
+          result.hasBlocks = true;
+        }
+      } else if (section.id.startsWith("political_group_votes_") && isFormSectionEmpty(section, values)) {
+        result.notableFormSections.push({
+          status: "empty",
+          formSection: section,
+          title: `Lijst ${section.id.substring(22)}`,
+        });
```

Comment:
If `Aantal kiezers en stemmen` is empty it should also be noted I think? I cannot find it in the design, but that would make sense to me. What do you think @jorisleker ?

Reply:
I already discussed this with Joris on mattermost.
Joris: "It's cleaner to only show lists"

Reply:
We discussed this about the "differences" page Les. 

If "aantal kiezers en stemmen" is empty, this results in a warning, so the case Ellen described can't exist.